docs/end-user-topics/dois.rst

```diff
@@ -69,10 +69,18 @@ Automatic Dockstore DOI Generation
 .. note::
     This feature is currently only available on request. Please contact the Dockstore team at `discuss.dockstore.org <https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506>`__ if you want to participate.
 
-* Automatically mints DOIs for all Git tags pushed to GitHub repositories that have a corresponding published entry in Dockstore
-* The DOIs are minted in the Zenodo Dockstore community
-* No Zenodo account is required to mint DOIs
-* A linked Zenodo account is required to edit DOIs
+Dockstore will automatically mint a DOI for a version that corresponds to a Git tag, if:
+
+* The entry is published on Dockstore
+* The version has at least one author in its metadata
+* The tag was created after the entry was published, or the tag is one of the 10 most recent
+
+Automatically-generated DOIs:
+
+* Do not require a Zenodo account
+* Are minted in the Zenodo Dockstore community
+* Are editable by request to the Dockstore team
+* Will be editable via a linked Zenodo account in a subsequent Dockstore release
```

Comment:
@kathy-t Should confirm, but I don't think the Zenodo account will need to be linked to Dockstore. I could be wrong, but I thought this feature will generate a signed Zenodo URL that anybody in possession of can edit with a Zenodo account.

Reply:
I removed "linked", we can add it back later if it turns out that it's accurate and important to note...

Reply:
Yeah that's correct, the user won't need to link their account to Dockstore, but they will need to sign into Zenodo after they click the secret Zenodo URL in order to edit the DOI record